.stylelintrc

```diff
@@ -9,6 +9,7 @@
         ]
       }
     ],
-    "no-descending-specificity": null
+    "no-descending-specificity": null,
+    "prettier/prettier": [true, {"singleQuote": false}]
```

Comment:
Thus is undoing the what is set in the config, because I had the config from a larger branch looking at Prettier across the different file types